src/Symfony/Component/Form/Resources/translations/sf_form.en.xlf

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0"?>
+<xliff version="1.2" xmlns="urn:oasis:names:tc:xliff:document:1.2">
+    <file source-language="en" datatype="plaintext" original="file.ext">
+        <body>
+            <trans-unit id="1">
+                <source>No choice available</source>
+                <target><em>No choice available.</em></target>
```

Comment:
its nothing specific to xliff. its about how xml namespaces work. when an xml parser parses the given xliff xml file, it interprets the em-element to be part of the xliff namespace (which is declared as the "default" namespace in the given document with `xmlns="urn:oasis:names:tc:xliff:document:1.2"`). This isnt a problem, unless the xliff consumer doesnt know what todo with an em-element. But xsd validation against the xliff schema will most likely fail (because it propably didnt define an em-element).

One way is to declare the xhtml namespace in the document ´xmlns:xh="http://www.w3.org/1999/xhtml"` and qualify the em-element against it: `<xh:em>...</xh:em>`. `http://www.w3.org/1999/xhtml` is used for xhtml 1.x and xml serialisation of html5 (xhtml5).

But since we dont care for the structure of the content in this use case, because the translation consumer expects `text/plain` produced by xliff, we can just properly escape the content of the target-element. either with `<target>&lt;em&gt;No choice available.$lt;/em$gt;</target>` or (more readable) `<target><![CDATA[<em>No choice available.</em>]]></target>` (cdata = character data)